android/res/layout/map_navigation_buttons.xml

```diff
@@ -9,29 +9,30 @@
   android:theme="?navButtonsTheme">
 
   <ImageButton
-    android:id="@+id/traffic"
-    style="@style/MwmWidget.MapButton.Traffic"
+    android:id="@+id/layers_button"
+    style="@style/MwmWidget.MapButton.Layers"
     android:layout_alignParentStart="true"
     android:layout_alignParentTop="true"
+    android:background="?toggleMapLayerBtnBg"
     android:visibility="invisible"
-    tools:src="@drawable/anim_traffic_loading"/>
+    tools:visibility="visible" />
 
   <ImageButton
-    android:id="@+id/subway"
-    style="@style/MwmWidget.MapButton.Traffic"
+    android:id="@+id/traffic"
+    style="@style/MwmWidget.MapButton.Layers"
     android:layout_alignParentStart="true"
     android:layout_alignParentTop="true"
     android:visibility="invisible"
-    android:background="?attr/nav_bg_subway"
-    tools:src="@drawable/anim_traffic_loading"/>
+    tools:src="@drawable/anim_traffic_loading" />
 
   <ImageButton
-    android:id="@+id/isolines"
-    style="@style/MwmWidget.MapButton.Traffic"
+    android:id="@+id/subway"
+    style="@style/MwmWidget.MapButton.Layers"
     android:layout_alignParentStart="true"
     android:layout_alignParentTop="true"
     android:visibility="invisible"
-    android:background="?nav_bg_isolines"/>
+    android:background="?attr/nav_bg_subway"
+    tools:src="@drawable/anim_traffic_loading"/>
```

Comment:
- Are this invisible traffic/subway buttons needed at all?
- Why did you replace isolines with traffic? We don't have traffic now, but have isolines.

Reply:
This was a temporary change. I have now refactored layers and there is only one layers button with the  `toggleMapLayerBtnBg` resource as background